packages/office-ui-fabric-react/package.json

```diff
@@ -47,6 +48,7 @@
     "node-sass": "^4.5.0",
     "office-ui-fabric-core": ">=5.0.0 <6.0.0",
     "postcss-loader": "^1.3.3",
+    "prop-types": "^15.5.9",
```

Comment:
Is that really? If so, why `react` is registered into devDependencies?

I only replaced `React.PropType` to `PropTypes`, I think its dependency was not changed essentially.